test/fixtures/flow/type-imports/invalid-import-type-shorthand/options.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'm probably just not familiar with how the test runner/fixtures work, but shouldn't this output have 2 throws (one for each line)?